src/submission/validation-clinical/specimen.ts

```diff
@@ -81,9 +81,36 @@ export const validate = async (
     checkTimeConflictWithDonor(donorDataToValidateWith, specimenRecord, errors);
   }
 
+  validatePercentTumour(specimenRecord, errors);
+
   return { errors };
 };
 
+const validatePercentTumour = (
```

Comment:
I see the pattern has already been established here before this PR, but just to document the though, in JavaScript it is considered unidiomatic to use/execute a function before it's been define and implemented.

i.e. this line 89 should be happening some line before 84, and the same goes for `checkTimeConflictWithDonor`, etc.

Reply:
Updated. All of the validation files are written this way. Makes me wonder how old the files are.